### ISSUE TITLE
New note not saved after install

### ISSUE THREAD

**Issue by ksigWyatt** [ThriveCommunityChurch/ThriveChurchOfficialApp#8]
## BUG

When a user first installs the app, and goes to the *Notes* tab, they will be prompted to return to the notes list in order for the app can save the note.

Once the user returns to the *Add Note* View the new note will be available to view in the list after exiting.

**Comment by ksigWyatt** [comment]
*Jumping in attempting to fix*

**Comment by ksigWyatt** [comment]
Not sure what to make of these logged errors from the console...
_Is that a socket exception? Maybe I'm missing something_

```
2017-08-26 21:42:30.513322 Thrive Church Official App [] nw_socket_write_close shutdown(18, SHUT_WR): [57] Socket is not connected
2017-08-26 21:42:30.514711 Thrive Church Official App [] nw_endpoint_flow_service_writes [9.1 192.0.76.3:443 ready socket-flow (satisfied)] Write request has 0 frame count, 0 byte count
2017-08-26 21:42:30.523562 Thrive Church Official App [] __tcp_connection_write_eof_block_invoke Write close callback received error: [89] Operation canceled
```

**Comment by ksigWyatt** [comment]
I'm increasing the estimate on this issue to as high as I can -  if not to infinity. I'm not sure what all effort it may take to fix this issue.
It might however be easier to refactor the whole note page again and use something different - Easier on me - so that I can fix issues easier if something else arises.

**Comment by ksigWyatt** [comment]
Preliminary testing yesterday showed that Branch `1.6` may have some kind of small fix for this issue. Will revisit.

**Comment by ksigWyatt** [comment]
The bug where the note is deleted has been fixed in `1.6`. The segue back to the `Table View` displaying the notes will trigger a new segue back to the *Add Note* screen. Where the text will still be stored. There is still some work to be done to perhaps prevent this segue from happening altogether. But the MAIN issue here is fixed for the following platforms. Testing the feature on other platforms still must be performed before I can say that this bug is fixed.

**Passing Devices / OS**
Devices:
- iPhone X
- iPhone 8 Plus
- iPhone 8
OS:
- iOS 11
